clearpath_robot/services/clearpath-manipulators.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description="Clearpath robot sub-service, launch all manipulators nodes"
+PartOf=clearpath-robot.service
+After=clearpath-robot.service
```

Comment:
Should this start after platform?

Reply:
There are no requirements for it to start after platform. The controllers are entirely independent.